### ISSUE TITLE
Confusion Matrix for far, near dataset for both training

### ISSUE THREAD

**Issue by deebuls** [RnDProjectsDeebul/SathwikPanchangamRnD#30]
Can you provide the confusion matrix for far and near dataset .

I wanted to check since the profiles are distance variant (from far F40 will look like F20) . It is visible in the confusion matrix.

**Comment by SathwikPanchangam** [comment]
Confusion matrix images for far and near constraints for Crossentropy, Evidential, Dropout and ensembles models.

[Confusion matrix for far and near constraint.pdf](https://github.com/RnDProjectsDeebul/SathwikPanchangamRnD/files/10529662/Confusion.matrix.for.far.and.near.constraint.pdf)
[confusion_matrix_distance.zip](https://github.com/RnDProjectsDeebul/SathwikPanchangamRnD/files/10529665/confusion_matrix_distance.zip)
